docker/puppetdb-postgres/Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM postgres:9.6.8
+ARG postgres_version="9.6.8"
+FROM postgres:"$postgres_version"
 
 ARG vcs_ref
 ARG build_date
 
+ARG postgres_version="9.6.8"
```

Comment:
I'm guessing you don't need this a second time.

Reply:
Yep, you do. Anything before the `FROM` is only in scope for the `FROM` and needs to be redeclared afterwards

Reply:
well that's.. silly.